### DISCUSSION TITLE
`vercel dev` is too slow for a simple hello world api

### DISCUSSION THREAD

**Discussion by goenning** [vercel/vercel#9598] [Help]
I have a simple one-file Node.js API using vercel serverless functions. I use `vercel dev` for local development and it takes ~500ms to respond to a simple hello world.

It seems like vercel CLI is compiling the code on every request? Can that be disabled or somehow cached?

**Comment by DevDavey** [comment]
I seem to have this same issue. On every single file serve it seems to be rebuilding. Did you manage to find the source of the problem?

**Reply by DevDavey** [reply]
I found the solution to my problem. For some reason, when my project contains a directory with many images (10,000+) every single file serve slows wayyy down. **Without** this large image directory files serve in ~100ms. **With** this directory in my project, it takes 10-30 seconds to serve any file from the server.

What's weird is that it slows down the serving of **every** file and not just the ones in the large directory. It ends up taking 1-3 minutes to load any page.

This only occurs in the dev environment. Everything is still fast deployed in production. I believe the slowdown has something to do with the resolveBuilders function although I am not certain. 

A temporary fix for me has been to add this image directory to my .vercelignore file. (And then remove it before deploying to production)
